Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/EmulationFragment.kt

```diff
@@ -98,8 +96,7 @@ class EmulationFragment : Fragment(), SurfaceHolder.Callback {
 
     override fun onResume() {
         super.onResume()
-        if (NativeLibrary.IsGameMetadataValid())
-            inputOverlay?.refreshControls()
+        if (NativeLibrary.IsGameMetadataValid()) inputOverlay?.refreshControls()
```

Comment:
We're moving away from braceless if statements in Kotlin. If you want to change these lines of code, then please add braces instead.